package.json

```diff
@@ -49,15 +50,14 @@
     "lint:server": "eslint ./server",
     "lint:web": "cd web && yarn lint",
     "prettier": "prettier \"**/*.{ts,tsx}\" --write",
-    "start": "yarn build:common && concurrently -p \"[{name}]\" -n \"CO,BE,FE\" -c \"bgYellow.bold,bgBlue.bold,bgMagenta.bold,bgCyan.bold\" \"cd common && yarn dev\" \"cd server && yarn start\" \"cd web && yarn dev\"",
-    "start:prod": "yarn build:common && node server/js/main.js",
+    "start": "yarn build:common && concurrently -p \"[{name}]\" -n \"CO,BE,FE\" -c \"bgYellow.bold,bgBlue.bold,bgMagenta.bold,bgCyan.bold\" \"cd common && yarn dev\" \"cd server && yarn start\" \"cd web && yarn dev\" \"yarn import-locales\"",
+    "start:prod": "yarn build:common && yarn import-locales && node server/js/main.js",
     "start:maint": "cd maintenance && yarn dev",
     "test": "yarn build:common && concurrently --kill-others-on-fail \"cd web && yarn test\" \"cd server && SERVER_CONFIG_PATH='../config.json' yarn test\""
   },
   "husky": {
     "hooks": {
       "pre-commit": "pretty-quick --staged"
     }
-  },
-  "packageManager": "yarn@1.22.18"
```

Comment:
Question: Why are we removing this? Seems useful as Yarn 1 and Yarn 2/3 are quite different

Reply:
Whoops, nice catch - didn't think I committed that!